docs/articles/documentation/guides/continuous-integration/github-actions.md

```diff
@@ -118,6 +118,8 @@ The TestCafe version to install.
 
 The following workflow demonstrates how to run TestCafe tests across Node.js versions and operating systems.
 
```

Comment:
How do I edit the above "Step" subtitles so that they match my edits to these subtitles?

Reply:
I'm afraid you can't, these lines were there before and I didn't modify them in this pull request. To edit them, another pull request is needed.

Or just tell me what needs to be  changed, I'll apply manually, thx

Reply:
Thanks, please make the following changes:
Line 14:
`* [Step 3 - Add a Step that Fetches the Repository](#step-3---add-a-step-that-fetches-the-repository)`
Line 15:
`* [Step 4 - Add a Step that Runs TestCafe](#step-4---add-a-step-that-runs-testcafe)`
Line 52:
`## Step 3 - Add a Step that Fetches the Repository`
Line 69:
`## Step 4 - Add a Step that Runs TestCafe`